src/tutorials/simple-service-provider.md

```diff
@@ -724,163 +718,70 @@ Add the following block of functions and variable into the `index.ts`:
-
-We'll be connecting to the Nym Websocket Client in the exact same way as the 'User Client', so we can reuse the following functions for the Service Provider: 
-* `sendSelfAddressRequest()`,
-* `display(message)`
-* `displayJsonResponse(message)`
-* `connectWebsocket(url)`
-
-#### Service Provider HTML Code
-
-Replace the current `index.html` with the following, to reflect our output on the UI for the Service Provider:
 
 ```
-    <!doctype html>
-    <html>
-        <head>
-            <style>
-                .headerContainer{
-                align-items: end;
-                display: flex;
-                margin-left: 10px;
-                margin-top: 10px;
-                }
-                .container{
-                    width: 400px;
-                    margin-left: 20px;
-                }
-            </style>
-            <meta charset="UTF-8">
-            <title>Mixnet Websocket Starter Service Provider</title>
-            <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/semantic-ui/2.5.0/semantic.min.css" integrity="sha512-KXol4x3sVoO+8ZsWPFI/r5KBVB/ssCGB5tsv2nVOKwLg33wTFP3fmnXa47FdSVIshVTgsYk/1734xSk9aFIa4A==" crossorigin="anonymous" referrerpolicy="no-referrer" />
-        </head>
-        <body>
-            <div class="headerContainer">
-            <img class="ui small image" src="https://nymtech.net/img/docs/FAVICON_DARK.png" style="height:45px;width:45px;">
-            <h1 style="margin-left: 10px;margin-top: 20px">Mixnet Websocket Starter Service Provider</h1> 
-            </div>
 
-            <h3 style="margin-left: 20px;">Recieved Message Data</h3>
-            <div class="ui list">
-            <div id="slideContainer"></div>
-            </div>
+> You may observe that we possess similar functions to the __User Client__ code, with only a few differences in the objectives of a few functions, mainly our `handleResponse()` function.
 
-            <hr>
+* `main()` - Just like our __User Client__, our `main()` function will still be the function in charge of our initializing and executing our application. We connect to our websocket in the exact same way as we do in our __User Client__ code except we want to set our `port` local variable to '1978'. This is so we don't have a conflict with the other Nym Websocket Client (the one that we are running for our __User Client__ on `port` 1977). So when we launch our second Nym Websocket Client, we will set the `--port` to 1978 when we get to initializing it (coming up further in the tutorial).
```

Comment:
Needs a reword, the grammar of the last sentence is off - seems to be a run on from the previous sentence, but is in fact a new one?